### PR TITLE
UPDATE dependencies, housekeeping

### PR DESCRIPTION
**Pull request by RecuencoJones**
I think we should wait for https://github.com/adidas/js-linter-configs/pull/25 before merging

### CODE REVIEW

File: examples/javascript-es5-browser/.eslintrc [hunk 1]
```diff
@@ -3,6 +3,9 @@
     "adidas-es5",
     "adidas-env/browser"
   ],
+  "parserOptions": {
+    "ecmaVersion": 6
```
Comment (moelders) [inline examples/javascript-es5-browser/.eslintrc:7]: Is it correct in `es5`?

File: package.json [hunk 2]
```diff
@@ -16,24 +16,27 @@
     "npm": ">= 5"
   },
   "scripts": {
-    "setup": "npm run lerna:bootstrap",
-    "lerna:bootstrap": "lerna bootstrap",
+    "setup": "npm run lerna:init && npm run lerna:bootstrap",
```
Comment (moelders) [inline package.json]: We can just run `lerna:bootstrap`.

File: package.json [hunk 3]
```diff
@@ -16,24 +16,27 @@
...
+    "lerna:init": "lerna init",
+    "lerna:bootstrap": "lerna bootstrap --no-ci",
     "lerna:exec": "lerna exec",
-    "lerna:publish": "lerna exec -- node \\$LERNA_ROOT_PATH/scripts/publish.js",
+    "lerna:publish": "lerna exec -- safe-publish",
+    "lerna:publish:dev": "npm run lerna:publish -- --registry=http://localhost:8081/repository/npm-local",
```
Comment (moelders) [inline package.json]: I would not push this.

**Review by moelders — CHANGES_REQUESTED** [review]

File: examples/javascript-es5-browser/.eslintrc (hunk 1 above)
Comment (RecuencoJones) [reply examples/javascript-es5-browser/.eslintrc:7]: It looks weird, but else `eslint` complains that we cannot use modules without `es2015` or greater.

Comment (moelders) [reply examples/javascript-es5-browser/.eslintrc:7]: OK.

**Review by moelders — APPROVED** [review]

**Comment by moelders** [comment]
LGTM

**Pull request merged**